pkgs/development/libraries/proj/default.nix

```diff
@@ -20,6 +21,12 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-Z2nruyowC3NG4Wb8AFBL0PME/zp9D7SwQdMSl6VjH/w=";
   };
 
+  patches = [ (fetchpatch {
+      url = "https://github.com/OSGeo/PROJ/commit/ac113a8898cded7f5359f1edd3abc17a78eee9b4.patch";
```

Comment:
```suggestion
      name = "fix-capi-test-open-max-limit.patch";
      url = "https://github.com/OSGeo/PROJ/commit/ac113a8898cded7f5359f1edd3abc17a78eee9b4.patch";
```

Reply:
Thanks for the suggestion, I'll use the commit message as the name like format-patch would.